api_app/views.py

```diff
@@ -574,3 +597,47 @@ class TagViewSet(viewsets.ModelViewSet):
     queryset = Tag.objects.all()
     serializer_class = TagSerializer
     pagination_class = None
+
+
+@add_docs(
+    description="""
+    REST endpoint to fetch list of CustomConfigs or retrieve/delete a CustomConfig.
+    Requires authentication. Allows access to only authorized CustomConfigs.
+    """
+)
+class CustomConfigViewSet(viewsets.ModelViewSet):
+    queryset = CustomConfig.objects.order_by("id")
+    serializer_class = CustomConfigSerializer
+    pagination_class = None
+
+    def get_queryset(self):
+        # Initializing empty queryset
+        result = CustomConfig.objects.none()
+
+        # Adding CustomConfigs for user's organization, if any
+        membership = Membership.objects.filter(
+            user=self.request.user, organization__isnull=False
+        )
+        if membership.exists():
+            result |= CustomConfig.objects.filter(
+                organization=membership[0].organization
+            )
+
+        # Adding CustomConfigs for user
+        result |= CustomConfig.objects.filter(owner=self.request.user)
```

Comment:
Why do you need to initialize this to an empty and use the `or` ? 
```suggestion
        # Adding CustomConfigs for user's organization, if any
        membership = Membership.objects.filter(
            user=self.request.user, organization__isnull=False
        )
        if membership.exists():
            result = CustomConfig.objects.filter(
                organization=membership[0].organization
            )
            else:

               result = CustomConfig.objects.filter(owner=self.request.user)
```

Reply:
Because the first `or` is conditional in nature, while the other isn't.

Reply:
Fuck, i can't even read today. You are 100% right